.github/workflows/windows-package.yml

```diff
@@ -52,3 +52,10 @@ jobs:
     - run:
           python setup.py bdist_wheel
       if: ${{ github.event_name == 'release' }}
+    - name: Install twine
+      run: python -m pip install twine
+    - name: upload wheels
+      env:
+          TWINE_USERNAME: __token__
+          TWINE_PASSWORD: ${{ secrets.TWINE_PASSWORD }}
+      run: twine upload  dist/* 
```

Comment:
Combine all lines from 52-61 into a single `run` block. All these commands must only run when a release is created.